contexts/DonationContext/tests/Unit/UseCases/AddDonation/AddDonationUseCaseTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Unit\DonationContext\UseCases\AddDonation;
+
+use ReflectionMethod;
+use PHPUnit\Framework\TestCase;
+use WMDE\Fundraising\Frontend\DonationContext\UseCases\AddDonation\AddDonationUseCase;
+
+/**
+ * @covers \WMDE\Fundraising\Frontend\DonationContext\UseCases\AddDonation\AddDonationUseCase
+ */
+class AddDonationUseCaseTest extends TestCase {
+
+	public function testGetInitialDonationStatus(): void {
+		$useCase = $this->createMock( AddDonationUseCase::class );
+		$method = new ReflectionMethod( $useCase, 'getInitialDonationStatus' );
+		$method->setAccessible( true );
```

Comment:
I do not think it is _better_, but another layer of security (the first actual unit test for this class). All this while the code is great overall and the existing tests are sweet.
The disadvantage of testing private methods this way is evident, but the flipside of having a class of 200 lines with only 2 public methods (which I do not oppose, just claim there is a price). I'd much rather have a unit test know the internals, the "how", of a class and have complete coverage of a method than having to craft n permutations of dummy object (we still do that _additionally_ for a few samples in integration, ...) to get anywhere close to the same level of confidence.
Concerning the "breaks automated refactoring": I'd actually want this test to break then.

Could that work for you?